base-theme/layouts/partials/featured_course_cards.html

```diff
@@ -23,8 +23,12 @@ <h2 class="{{ $klass }}">Featured Courses</h2>
             {{ range $index, $courseItem := $courses }}
               {{- $urlPath := partial "site_root_url.html" $courseItem.id -}}
               {{- $url := delimit (slice (strings.TrimSuffix "/" $staticApiBaseUrl) $urlPath "/data.json") "" -}}
-              {{- $courseData := getJSON $url -}}
-              {{ partial "course_carousel_card.html" (dict "itemsInCarousel" $itemsInCarousel "courseData" $courseData "index" $index "urlPath" $urlPath "numCourses" (len $courses))}}
+              {{ with resources.GetRemote $url }}
+                {{ if not .Err }}
```

Comment:
@gumaerc Do you have any suggestion of which approach to take in order to log an error to Sentry from here in HTML? 
Do we have anything in Hugo that can help us with it?
I think we have `errorf` and `warnf` in Hugo but both will cause the build to fail